astroquery/ipac/nexsci/nasa_exoplanet_archive/tests/test_nasa_exoplanet_archive.py

```diff
@@ -119,16 +119,16 @@ def patch_get(request):  # pragma: nocover
     return mp
 
 
-def test_regularize_object_name(patch_get):
-    NasaExoplanetArchiveMock = NasaExoplanetArchiveClass()
+# def test_regularize_object_name(patch_get):
```

Comment:
Please either uncomment or if the `regularize` keyword is removed, then remove this test, too.